server/routes/api/user.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""User related handlers."""
+
+import flask
+import json
+
+from flask import current_app
+
+bp = flask.Blueprint('api.user', __name__, url_prefix='/api/user')
+
+
+def get_user_info(google_id):
+    """Returns data given a user id."""
+    db = current_app.config['USER_DB']
+    users_ref = db.collection(u'users')
```

Comment:
Could we index "users" by hash(google_id) to not loop?

Reply:
Done